force-app/main/default/classes/FileUtilities.cls

```diff
@@ -0,0 +1,33 @@
+public with sharing class FileUtilities {
+    @AuraEnabled
+    public static String createFile(
+        String base64data,
+        String filename,
+        String recordId
+    ) {
+        try {
+            ContentVersion contentVersion = new ContentVersion();
+            contentVersion.VersionData = EncodingUtil.base64Decode(base64data);
+            contentVersion.Title = filename;
+            contentVersion.PathOnClient = filename;
+            insert contentVersion;
```

Comment:
```suggestion
            insert as user contentVersion;
```

Reply:
I had this, but removed it because I couldn't commit - any updates on the eslint plugin fix?

Reply:
Let's proceed without this for now (see my Slack message).